content/en/_index.md

```diff
@@ -275,22 +308,66 @@ async function listInfoTypes() {
   });
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+import (
+	"context"
+	"fmt"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// listInfoTypes lists the types of sensitive information the DLP API supports.
+func listInfoTypes(w io.Writer) {
+	ctx := context.Background()
+	// Initialize client that will be used to send requests. This client only
+	// needs to be created once, and can be reused for multiple requests. After
+	// completing all of your requests, call the "Close" method on the client to
+	// safely clean up any remaining background resources.
+	c, err := dlp.NewClient(ctx)
+	if err != nil {
+		// TODO: Handle error.
+	}
+	defer c.Close()
+
+	// Construct the request to be sent by the client
+	req := &dlppb.ListInfoTypesRequest{
+		// Only return infoTypes supported by certain parts of the API.
+		// Supported filters are "supported_by=INSPECT" and "supported_by=RISK_ANALYSIS"
+		// Defaults to "supported_by=INSPECT"
+		Filter: "supported_by=INSPECT",
+		// BCP-47 language code for localized infoType friendly names.
+		// Defaults to "en_US"
+		LanguageCode: "en-US",
+	}
+
+	resp, err := c.ListInfoTypes(ctx, req)
+	if err != nil {
+		// TODO: Handle error.
```

Comment:
This needs a return statement even if we are not returning errors. Same in L329 above